File: Frontend/src/app/system/_domains/StockSum.ts
import { Injectable } from '@angular/core';
import { _EntityBase } from './_EntityBase';

@Injectable()
export class StockSum extends _EntityBase
{
     public IdProduct: number;
     public ProductName: string;
     public Brand: string;
     public Picture: string;
     public InternalNumber: string;
     public BarCode: string;
     public Quantity: number;

}

File: Backend/Stock.Business/Entities/StockSum.cs
using Stock.Domain;
using System;
using System.Collections.Generic;
using System.Text;

namespace Stock.Business.Entities
{
    public class StockSum
    {
        public int Id { get; set; }
        public int IdProduct { get; set; }
        public string ProductName { get; set; }
        public string Brand { get; set; }
        public string Picture { get; set; }
        public string InternalNumber { get; set; }
        public string BarCode { get; set; }
        public decimal Quantity { get; set; }
        public DateTime CreateDate { get; internal set; }
    }
}
